gitpod-ws.code-workspace

```diff
@@ -8,6 +8,7 @@
         { "path": "components/ee/agent-smith" },
         { "path": "components/gitpod-cli" },
         { "path": "components/gitpod-protocol" },
+        { "path": "components/ide-proxy/components/ide-metrics-server" },
```

Comment:
I think we can put it in `components/ide-metrics-server` similarly to other components and maybe just name `ide-metrics`

Reply:
the same for `components/ide-metrics-api`